src/components/Agreed/Agreed.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Paragraph } from 'grommet';
+
+import { spacingLarge } from '../../styles/variables';
+
+const Wrapper = styled.section`
+  margin: ${spacingLarge} auto;
+`;
+
+const Agreed = function() {
```

Comment:
This is the comment related to the whole project.
You are mixing different ways of function declaration. Define some rules for that
1. Either u use arrow function with variable, or simple function declaration.
`const Agreed = () => {...} or function Agreed() {...}`
Don't do something like that
`const Agreed = function() {...}`
2. Define rules. If you use arrow function on the very first level, use it everywhere. Don't mix different declarations on the same level functions. If you don't know what to choose, pick arrow functions everywhere.
3. If you use arrow function that only returns some data, omit the `return` keyword.